test/es-module/test-esm-import-meta.mjs

```diff
@@ -0,0 +1,7 @@
+// Flags: --experimental-modules --harmony-import-meta
+
+import '../common';
+import assert from 'assert';
+
+const urlReg = /^file:\/\/\/.*\/test\/es-module\/test-esm-import-meta\.mjs$/;
+assert(import.meta.url.match(urlReg));
```

Comment:
per https://github.com/nodejs/node/pull/18368#pullrequestreview-91595941 we can add
```mjs
assert.stictEqual(Object.getPrototypeOf(import.meta), null);
assert.deepEqual(Object.keys(Object.getOwnPropertyDescriptors(import.meta)), ['url']);
```

Reply:
Could the second line use `Reflect.ownKeys(import.meta)` instead?

Reply:
seems fine

Reply:
change `['urls']` to `[{ enumerable: true, writable: false, configurable: false, value: 'url' }]`, eg

Reply:
`Reflect.ownKeys` is good because it also covers symbols; but it doesn't cover descriptors.